ee/clickhouse/queries/funnels/base.py

```diff
@@ -178,9 +178,10 @@ def _format_single_funnel(self, results, with_breakdown=False):
 
             serialized_result.update(
                 {
-                    "converted_people_url": f"{self._base_uri}api/person/funnel/?{urllib.parse.urlencode(converted_people_filter.to_params())}",
-                    "dropped_people_url": (
-                        f"{self._base_uri}api/person/funnel/?{urllib.parse.urlencode(dropped_people_filter.to_params())}"
+                    "actor_endpoint": "api/person/funnel/",
+                    "converted_actor_params": f"{urllib.parse.urlencode(converted_people_filter.to_params())}",
```

Comment:
We'll have issues with backwards compat. and cached responses I think.

Reply:
yeah, I might do this a different way to save us the trouble right now